pyrate/core/gamma.py

```diff
@@ -17,16 +17,15 @@
 This Python module contains tools for reading GAMMA format input data.
 """
 # coding: utf-8
-
 from os.path import join, split
 import re
 import os
-import glob2
 from datetime import date, time, timedelta
 import numpy as np
 import pyrate.core.ifgconstants as ifc
 from pyrate.core import config as cf
 
+
 PTN = re.compile(r'\d{8}')  # match 8 digits for the dates
```

Comment:
I am concerned if more regex creeps in we would lose track of the reason for using these patterns. Can we move this to either constant file and import it here or expand the name to something on the line of "regex pattren 8 digits" and update comments to 8 digits represent the epoch time in the file name.